frontend/projects/admin/src/app/event/translate-payment-proxies.pipe.ts

```diff
@@ -0,0 +1,22 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+const PAYMENT_PROXY_DESCRIPTIONS = {
+  STRIPE: 'Stripe: Credit cards',
+  ON_SITE: 'On site (cash) payment',
+  OFFLINE: 'Offline payment (bank transfer, invoice, etc.)',
+  PAYPAL: 'PayPal',
+  MOLLIE:
+    'Mollie: Credit cards, iDEAL, Bancontact, ING Home Pay, Belfius, KBC/CBC, Przelewy24',
+  SAFERPAY: 'Saferpay By SIX Payments',
+} as any;
+
+@Pipe({
+  name: 'translatePaymentProxies',
+})
+export class TranslatePaymentProxiesPipe implements PipeTransform {
+  transform(list: string[], ...args: unknown[]): unknown {
```

Comment:
- remove additional unused argument "args"
- this method should return string[] instead of unknown